hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/DiskBalancerService.java

```diff
@@ -352,6 +359,10 @@ public BackgroundTaskQueue getTasks() {
     }
 
     if (queue.isEmpty()) {
+      bytesToMove = 0;
+      if (stopAfterDiskEven) {
+        setShouldRun(false);
```

Comment:
Please add a info log to tell that disk balancer is stopped due to disk even and this property true.

Besides change the in memory shouldRun flag, also need to persist the status into yaml file. Otherwise, next DN restart, disk balancer will start to run again.